src/components/molecules/Visualizer/Engine/index.tsx

```diff
@@ -53,6 +53,13 @@ export type SceneProperty = {
     enableGA?: boolean;
     trackingId?: string;
   };
+  reTheme?: {
+    reThemeMode?: "preMadeTheme" | "customTheme";
+    preMadeThemeId?: "reEarthLight" | "reEarthDark" | "forest";
+    textColor?: string;
+    selectColor?: string;
+    backgroundColor?: string;
+  };
```

Comment:
```suggestion
  theme?: {
    themeType?: "light" | "dark" | "forest" | "custom";
    themeTextColor?: string;
    themeSelectColor?: string;
    themeBackgroundColor?: string;
  };
```

Refer to https://github.com/reearth/reearth-backend/pull/35#discussion_r694462910